documentation/publish/package-your-extension.md

```diff
@@ -18,6 +18,7 @@ contributors:
     david_ross,
     wbamberg,
     mrj,
+    mickro,
   ]
 last_updated_by: potterwrit
 date: 2019-07-15 1:14:39
```

Comment:
Could you please update these lines as well? Thanks

Reply:
which TZ are you using?

Reply:
umm not sure , may be @caitmuenster can answer , but you can remove the time just date would be fine. Thanks

Reply:
should be ok now

Reply:
That's good. Thanks @mickro! :)